css/main.css

```diff
@@ -211,3 +211,30 @@ section .read-more-btn:hover svg path {
 }
 /* End of Resources Section */
 /* ================================== */
+/* Blog Section */
+#blog .img-container {
+  overflow: hidden;
+  cursor: pointer;
+}
+#blog .img-container:hover .caption {
+  bottom: 0;
+  left: 0;
+  right: 0;
+  opacity: 1;
+}
+#blog .caption {
+  background-color: rgba(255, 255, 255, 0.8);
+  z-index: 1;
+  bottom: -10px;
+  min-height: 110px;
+  opacity: 0;
+  transition: all .25s ease-in-out;
+}
+.position-abs {
```

Comment:
Bootstrap 4 has classes `.position-absolute` and `.position-relative`. There is no need to recreate it in `main.css` file